### ISSUE TITLE
t/version.t fails

### ISSUE THREAD

**Issue by eserte** [perl5-dbi/DBD-mysql#362]
On all of my smokers (if the compilation was successful) the test suite fails:
```
#   Failed test 'version starts with "4." (update for 5.x)'
#   at t/version.t line 9.
#                   '5.001'
#     doesn't match '(?-xism:^4\.)'
# Looks like you failed 1 test of 3.
t/version.t ............................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/3 subtests 
```

See also http://matrix.cpantesters.org/?dist=DBD-mysql%205.001 : currently there are no pass reports.

**Comment by dakkar** [comment]
also:
```
# mysql_clientinfo is: 5.7.33
# mysql_clientversion is: 50733
# mysql_serverversion is: 50651
# mysql_hostinfo is: Localhost via UNIX socket
# mysql_serverinfo is: 5.6.51-91.0
# mysql_stat is: Uptime: 1028  Threads: 1  Questions: 1626  Slow queries: 0  Opens: 215  Flush tables: 1  Open tables: 80  Queries per second avg: 1.581
# mysql_protoinfo is: 10
# SQL_DBMS_VER is 5.6.51-91.0

#   Failed test 'SQL_DRIVER_VER starts with "04." (update for 5.x)'
#   at t/10connect.t line 48.
#                   '05.01.0000'
#     doesn't match '(?^:^04\.)'
# Default storage engine is: InnoDB
# Looks like you failed 1 test of 15.
t/10connect.t ........................... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/15 subtests
```

**Comment by dveeden** [comment]
This was fixed in #364
